### PR TITLE
for code review from michael

### PR DESCRIPTION
**Pull request by styvone**
i'm getting `ReferenceError: chrome is not defined` on line 57 of `storageController.js`

### CODE REVIEW

File: src/utils/storageController.test.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import StorageController from "./storageController";
+
+describe("StorageController", () => {
+  const fakeChrome = {
+    storage: {
+      local: "local",
+      sync: "sync"
+    }
+  };
```
Comment (michaelnyu) [inline src/utils/storageController.test.js]:
```js
const fakeStorageStrategy = {
  storage: {
    set: jest.fn(),
    remove: jest.fn(),
    ...
  }
}
```

File: src/utils/storageController.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+  test("sets shortcuts correctly without safety check", () => {
+    const storageController = new StorageController(fakeChrome.storage);
```
Comment (michaelnyu) [inline src/utils/storageController.test.js]: `const storageController = new StorageController(fakeStorageStrategy);`

File: src/utils/storageController.test.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    const links = [];
+    storageController.setShortcut("https://www.google.com", links);
+
+    // test that shortcuts were set correctly
+    expect(storageController.getShortcutURLs("https://www.google.com")).toEqual(
+      links
+    );
```
Comment (michaelnyu) [inline src/utils/storageController.test.js]: not sure on exact syntax

```js
storageController.getShortcutURLs("https://www.google.com");
expect(fakeStorageStrategy.storage.get).to.have.been.calledOnce();
```

Comment (styvone) [reply src/utils/storageController.test.js]: Can you explain this more ?

Comment (michaelnyu) [reply src/utils/storageController.test.js]: we are mocking the function using `jest.fn()` and then ensuring it is being called in our test.

**Pull request merged**